pilz_msgs/srv/BrakeTest.srv

```diff
@@ -0,0 +1,21 @@
+#
+# Copyright (c) 2019 Pilz GmbH & Co. KG
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+---
+bool success
+
+# Contains message explaining failure of the brake test execution
+string error_msg
```

Comment:
I accept this for now but only under the condition that we make a separate meeting in which we discuss a guideline on how-to properly transmit errors from ROS to the PythonAPI. This is a general issue which needs to be discussed. In my opinion, it should be consistent for all messages, etc. which are exchanged between ROS and the PythonAPI.